terraform/rds.tf

```
@@ -33,11 +33,15 @@ resource "aws_db_subnet_group" "grafana" {
   }
 }
 
+data "aws_secretsmanager_secret_version" "grafana_db_backend" {
+  secret_id = "grafana-backend-db-creds"
+}
+
```

thinking we should be able to have different passwords for the different downstream envs, so like 
```suggestion
data "aws_secretsmanager_secret_version" "grafana_db_backend" {
  secret_id = format("%s-grafana-backend-db-creds", var.env)
}
```

---

True, addressed - I guess in theory shouldn't matter now that we have (or will have) separate accounts for each env but good to stay consistent.